pom.xml

```diff
@@ -330,6 +330,27 @@ under the License.
 				</executions>
 			</plugin>
 
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>2.17</version>
+				<executions>
+					<execution>
+						<id>test</id>
+						<phase>test</phase>
+						<configuration>
+							<configLocation>style/checkstyle.xml</configLocation>
```

Comment:
w.r.t. foldername: 
i'd prefer naming after function and not implementation, but not really an issue to.
Is the default eclipse formatter used by everyone? Custom files might go here as well.